swift/perfect/Package.swift

```diff
@@ -4,9 +4,9 @@ import PackageDescription
 let package = Package(
     name: "server",
     dependencies: [
-        .package(url: "https://github.com/PerfectlySoft/Perfect-HTTPServer.git", "3.0.0"..<"3.1.0")
+        .package(url: "https://github.com/PerfectlySoft/Perfect", "3.1.0"..<"3.2.0")
```

Comment:
```suggestion
        .package(url: "https://github.com/PerfectlySoft/Perfect", .upToNextMinor(from: "3.1.0"))
```

The more idiomatic way of specifying version ranges for SwiftPM is using the available requirement type. Both work fine though 😊